### ISSUE TITLE
How to test my own video?

### ISSUE THREAD

**Issue by fengtingliang** [zlai0/MAST#10]
Thanks for your sharing. And I would like to know whether I can test my own video?

**Comment by zlai0** [comment]
Yes. You need write your own data loader (use `functional/feeder/dataset/Davis2017.py` as a reference).
